src/main/java/core/basesyntax/service/FileReadImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileRead;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReadImpl implements FileRead {
+    @Override
+    public List<String> read(String fileName) {
```

Comment:
The parameter `fileName` should be checked for null or empty values to avoid potential `FileNotFoundException` or reading an unintended file. You can throw an IllegalArgumentException if the `fileName` is not valid.